src/storage/src/hummock/store/version.rs

```diff
@@ -139,7 +139,23 @@ impl HummockReadVersion {
             }
 
             VersionUpdate::CommittedSnapshot(committed_version) => {
+                let max_committed_epoch = committed_version.max_committed_epoch();
                 self.committed = committed_version;
+
+                // TODO: remove it when support update staging local_sst
+                self.staging
+                    .imm
+                    .retain(|imm| imm.epoch() > max_committed_epoch);
+                self.staging.sst.retain(|sst| {
+                    sst.epochs.first().expect("epochs not empty") > &max_committed_epoch
+                });
+
+                #[cfg(debug_assertions)]
+                {
```

Comment:
In the `new` of StagingSstableInfo we can add an assert that the epochs are sorted from higher epoch to lower epoch. And here we may also need to assert epochs.last() > MCE. since this costs little, we can enable this assert not only in the debug_assertion.